book/src/generic-builds.md

```diff
@@ -0,0 +1,80 @@
+# Generic Builds
+
+> since 0.5.0
+
+Although cargo-dist was originally designed specifically for Cargo-based builds, we've introduced experimental features to allow you to use it to build and distribute any software written in any language. This feature is currently a *prerelease*: details may change before it becomes stable.
```

Comment:
> This feature is currently a *prerelease*: details may change before it becomes stable.

Is this a "change this line before cutting the release" or "this is unstable, even past the release"

Reply:
We're considering the feature experimental at this point, so it's still the case that we might change some of these details at some point.